examples/temperature.rs

```diff
@@ -0,0 +1,52 @@
+//! A simple libtock-rs example. Checks for temperature driver
+//! and samples the sensor every 2 seconds.
+
+#![no_main]
+#![no_std]
+
+use core::fmt::Write;
+use libtock::console::Console;
+
+use libtock::alarm::{Alarm, Milliseconds};
+use libtock::runtime::{set_main, stack_size};
+use libtock::temperature::Temperature;
+
+set_main! {main}
+stack_size! {0x200}
+
+fn main() {
+    match Temperature::exists() {
+        Ok(()) => writeln!(Console::writer(), "temperature driver available").unwrap(),
+        Err(_) => {
+            writeln!(Console::writer(), "temperature driver unavailable").unwrap();
+            return;
+        }
+    }
+
+    loop {
+        match Temperature::read_temperature_sync() {
+            Ok(temp_val) => {
+                if temp_val >= 0 {
+                    writeln!(
+                        Console::writer(),
+                        "Temperature: {}.{}*C\n",
+                        temp_val / 100,
+                        temp_val % 100
+                    )
+                    .unwrap()
+                } else {
+                    writeln!(
+                        Console::writer(),
+                        "Temperature: -{}.{}*C\n",
+                        (0 - temp_val) / 100,
+                        (0 - temp_val) % 100
+                    )
+                    .unwrap()
+                }
```

Comment:
```suggestion
                    writeln!(
                        Console::writer(),
                        "Temperature: {}.{}*C\n",
                        temp_val / 100,
                        i32::abs(temp_val) % 100
                    )
                    .unwrap()
```

Reply:
That suggestion would format a `temp_val` of `-50` (i.e. -0.5 degrees) as `0.50`.

Reply:
@alexandruCarp any update?

Reply:
Well, as said above, that suggestion would still produce incorrect results for values in range (-1,0), as no expression could produce "minus 0" as its result.

Reply:
Didn't see the observation, I have updated the suggestion.